docs/secure-connections.md

```diff
@@ -51,37 +53,37 @@ Our IPSec VPN solution establishes a VPN connection between two IPSec gateways,
   <div className="box-wrapper" markdown="1">
     <div className="box box1 card">
       <div className="container">
-      <h3><a href="https://wiki.saucelabs.com/pages/viewpage.action?pageId=48365718">Using Sauce Connect Proxy</a></h3>
-      <p>Review Sauce Connect system and network requirements then install the Sauce Connect Proxy client.</p>
+      <h3><a href="/secure-connections/sauce-connect">Using Sauce Connect Proxy</a></h3>
+      <p>Review the Sauce Connect Proxy system and network requirements, and then install the Sauce Connect Proxy client.</p>
       </div>
     </div>
     <div className="box box2 card">
       <div className="container">
-      <h3><a href="https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+Setup+and+Configuration">Setup and Configuration</a></h3>
+      <h3><a href="/secure-connections/sauce-connect/setup-configuration/setup-configuration">Setup and Configuration</a></h3>
```

Comment:
is this right?  `/secure-connections/sauce-connect/setup-configuration/setup-configuration`

Reply:
Yes!